src/client/components/PostAct/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+class PostAct extends React.Component {
+  render() {
+    return (
+      <div className='postAct'>
+        {/* <button className='likeButton'>    Like</button> */}
+        <button className='likeButton' type="button">
+          <img className='likeIcon' src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAB2klEQVRoQ+2Z0U3DMBCGz7oBKBu0G9Tq+R0mKGwAEwAblA1gA7oBTEB5txQ2oGzQvjcyspRKkDpN7VyLT0qeHfv/7j/bl4sC4Y8Srh96gP928GQOFEUxKMtySkRzTuiTAHjxm83mXSk1BoAlIl5rrT85QI4OUBO/1bxCRK21XnaFOCpAg/it5jkR3WQL0CIenHMfxpiLLAHaxHvR2QIcIr6K+jMR3WflQIR478ClMWaRDUCMeABYE9Ggq3j/PsspFCner8tyArEAJIhnS5/OACniAeCbiIYc6dMJIFG8X5Pl9NkGIHkPWGufAOAuNpJVCcFSBwUdsNa6kCgi+gNrrf0CALZU2BMIXy8tEPFBa72qj9txIAIgCBrrSMR4XwCO6hCSAILHrzSAJRGNfrsmDQDqe7EHiNiELEOlO/BGRFdi94BS6nYymbxIBQiW4GI2sXPu0RgzE3sTI+K55FKi8QNIRApVNVCwCZY9QFv7RQLA3u5F7gCtn59ZA4QuLknH6EG9o2wdaLq4xDjQdHFxArwCwJSlRt6d5ODOXXIKFUUxLsuyOALAGhGHobIhtFYygJ+sgvAFlv9RccYAM0fEWcyvp+TGFoNYlil6AJYwdpikd6BD8FheFe/AD0dYZkBkaW/+AAAAAElFTkSuQmCC"/>
```

Comment:
this img is not necessary., you can use background image instead. If you remove the image, you dont need to use the span neither.